samples/otlptracegrpc/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This looks a bit different from the ones GTS generated https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/blob/04bd532e91387b070d5c7b785f56c411dde2dc91/packages/opentelemetry-resource-util/tsconfig.json#L1-L11


Did GTS add the `esModuleInterop: true` recently? I think they also recommend keeping sources in a `src` folder

Reply:
No, I added that, since I required that for this sample.

Yes, but seeing as none of the other samples are in a `src` folder - I found that to be an overkill for this.
I actually tried the `npx gts init` but I had a hard time configuring and adapting it to the current sample structure.